src/tests/healthcareProfessionalQuery.ts

```diff
@@ -23,28 +25,32 @@ describe('GraphQL', () => {
                 if (err) { return done(err); }
 
                 console.log(res.body);
-                expect(res.body.data.healthcareProfessional).to.have.property('id');
+                expect(res.body.data.healthcareProfessional).to.have.property('id').equal('1');
                 expect(res.body.data.healthcareProfessional).to.have.property('names');
 
                 const { names } = res.body.data.healthcareProfessional;
 
-                expect(names[0]).to.have.property('firstName');
+                expect(names[0]).to.have.property('firstName').equal('Larissa');
                 done();
 
                 return true;
             });
     });
 
-    //   it("Returns all users", (done) => {
-    //     request
-    //       .post("/graphql")
-    //       .send({ query: "{ user { id name username email } }" })
-    //       .expect(200)
-    //       .end((err, res) => {
-    //         // res will contain array of all users
-    //         if (err) return done(err);
-    //         // assume there are a 100 users in the database
-    //         res.body.user.should.have.lengthOf(100);
-    //       });
-    //   });
+    it('Returns all users', done => {
+        request
+            .post('/graphql')
+            .send({ query: '{ healthcareProfessionals { id } }' })
+            .expect(200)
+            .end((err:any, res: any) => {
```

Comment:
We should use async/await here instead of callbacks like `end`

Reply:
Let's refactor later, this was just the example. 

We had no testing, now we have something. Feel free to open a ticket

Reply:
(If you look closely at the diff you'll see I just uncommented the example and fixed it)

Reply:
ok sounds good! I'll create a separate ticket

and thanks again for writing tests ^_^